arrow-fx-coroutines/src/main/kotlin/arrow/fx/coroutines/stream/TerminalOps.kt

```diff
@@ -106,3 +134,6 @@ suspend fun <O, B> Stream<O>.foldChunks(init: B, f: (B, Chunk<O>) -> B): B =
 
 private suspend fun <O, B> Stream<O>.compiler(init: B, foldChunk: (B, Chunk<O>) -> B): B =
   asPull.compiler(init, foldChunk)
+
+@Suppress("NOTHING_TO_INLINE") // we want to avoid polluting the stack trace
```

Comment:
Does this work to not pollute the stack trace?

Reply:
I think that, from memory, inline functions don't show up on the stack trace... Maybe should add some tests for this 🤔